js/VimeoView.js

```diff
@@ -32,7 +32,7 @@ export default class VimeoView extends Backbone.View {
    */
   setupPlayer() {
     const options = {
-      url: this.model._source,
+      url: this.model._source.trim(),
```

Comment:
```suggestion
      url: this.model._source?.trim(),
```
You may want to add the optional chaining operator in case the _source object isn't defined

Reply:
Hi Paul, if you use the 'commit suggestion' button above, I think this PR is then ready